static/src/javascripts/boot.js

```diff
@@ -29,17 +29,26 @@ define([
     // curl’s promise API is broken, so we must cast it to a real Promise
     // https://github.com/cujojs/curl/issues/293
     var promiseRequire = function (moduleIds) {
-        return Promise.resolve(require(moduleIds));
+        return Promise.resolve(window.require(moduleIds));
     };
-
     var guardian = window.guardian;
     var config = guardian.config;
 
     var domReadyPromise = new Promise(function (resolve) { domReady(resolve); });
 
     var bootStandard = function () {
-        return promiseRequire(['bootstraps/standard/main'])
-            .then(function (boot) { boot(); });
+        if (config.tests.abWebpackBundle) {
+            return new Promise(function (resolve) {
+                require(['bootstraps/standard/main'], function (boot) {
+                    boot();
+                    resolve();
+                });
+            });
+        }
+        else {
```

Comment:
don't need this else (eslint has been picking me for months on this in other parts of the codebase 😉 )